content/blog/2023/02/2023-02-05-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,133 @@
+
+Contributed by: author:dduportal[Damien Duportal]
+
+* Bumped our controllers to LTS 2.375.2
+* General availability of new development tools on ci.jenkins.io:
+** Maven 3.8.7
+** JDK 8u362-b09
+** JDK 11.0.18+10
+** JDK 17.0.6+10
+** JDK 19.0.2+7
+** Playwright (headless web-browser testing)
+
+* Started to decrease download bandwidth from JFrog (repo.jenkins-ci.org)  to our infrastructure by enabling artifact caching for plugins builds on  ci.jenkins.io
+** A lot of “under-the-hood” network and performances related work
+* Cleaned up deprecated plugins from all of our controllers (momentjs,  jquery bundle and ace-editor bundle)
+
+
+[[documentation]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/jenkins-io-improvements.png[Jenkins io improvements Update]
+
+Contributed by: author:kmartens27[Kevin Martens]
+
+During the month of January, 98 pull requests were merged from 38  different authors.
+We are also preparing for the 2023 edition of Google  Summer of Code, by encouraging folks to share ideas or join as mentors.
+We published one blog post, which was the 2022 recap and contained tons
+of highlights for the last year of the Jenkins project.
+Thanks to all of the new and returning contributors for their hard work already in the new year, and the continued efforts that are bound to happen.
+
+[[Governance]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/governance.png[Governance Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+Jenkins 2.375.2 released January 11, 2023.
+Over 350 positive https://www.jenkins.io/changelog-stable/#v2.375.2[ratings] with only 1  issue reported to have required a rollback for 4 users.
+
+The most recent 9 Jenkins weekly releases (2.381 - 2.389) have received
+positive https://www.jenkins.io/changelog/#v2.389[ratings] as well,
+with 40-50 positive ratings and only 1 reported rollback in each of
+those 9 weekly releases.
+
+Thanks to everyone that has done such great work on Jenkins core and its
+recent releases.
+
+[[platform]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/platform-modernization.png[Platform Modernization Update]
+
+Contributed by: author:gounthar[Bruno Verachten]
+
+* Docker images
+** New platforms:
+*** UBI9 with JDK17 proposed and maintained by Oliver Gondza from RedHat
+(also maintainer of UBI8 container image)
+*** https://github.com/jenkins-infra/helpdesk/issues/3318[Zombie
+images] fixed. Hurray!
+*** Java updates were deployed to infra and container images (except for
+Windows containers)
+**** 8u362
+**** 11.0.18
+**** 17.0.6
+**** 19.0.2
+* Debian 12 (“bookworm”) will not deliver OpenJDK 11
+** End of life date for Debian’s openJDK11 won’t happen until 2026 or
+2027
+** No urge to drop jdk11 support for Jenkins
+** Will change the documentation nonetheless when it goes out, so that  we describe the use of Jenkins with openJDK17
+* JDK Support for Jenkins
+** Jenkins' enhancement proposal for required JDK11 is final! Congrats.
```

Comment:
```suggestion
*** UBI9 with JDK17 proposed and maintained by Oliver Gondza from RedHat, who's also maintainer of the UBI8 container image.
*** https://github.com/jenkins-infra/helpdesk/issues/3318[Zombie images] fixed. Hurray!
*** Java updates were deployed to infra and container images (except for Windows containers):
**** 8u362
**** 11.0.18
**** 17.0.6
**** 19.0.2
* Debian 12 (“bookworm”) will not deliver OpenJDK 11.
** The end of life date for Debian’s openJDK11 won’t happen until 2026 or 2027.
** There is no urge to drop jdk11 support for Jenkins.
** We will change the documentation nonetheless when it goes out, so that  we describe the use of Jenkins with openJDK17.
* JDK Support for Jenkins
** Jenkins' enhancement proposal for required JDK11 is final!
Congrats.
```
single sentence per line and full stop suggestions